### ISSUE TITLE
Can someone tell me what happened?

### ISSUE THREAD

**Issue by Gromah** [Maxx53/SCMBot#481]
I don't get what happened recent days 'cause everything is in cyrylic. In short something happened to SCM? Should I move to SMBot or what?

**Comment by Maxx53** [comment]
no, something happend with steam community servers. you can try other bots to make sure.

**Comment by mtzi** [comment]
Steam is getting DDOSed every fkin day, theres downtime, and ppl lose their shit, i still get my buys-sells normally. Of course when Steamcommunity is dead i get errors. but its normal
